test/dotcom_web/controllers/transit_near_me_controller_test.exs

```diff
@@ -161,42 +99,77 @@ defmodule DotcomWeb.TransitNearMeControllerTest do
 
   describe "with valid location params" do
     test "assigns stops with routes", %{conn: conn} do
-      params = %{"address" => %{"latitude" => "valid", "longitude" => "valid"}}
+      stops = build_list(3, :stop_item)
 
-      conn = get(conn, transit_near_me_path(conn, :index, params))
+      expect(LocationService.Mock, :reverse_geocode, fn latitude, longitude ->
```

Comment:
I *think* you can just `@latitude` and `@longitude` here. If not, you *might* be able to pin them.